events/uk/cbcd_contra

```diff
@@ -0,0 +1,35 @@
+- name: Contra Dance in Cambridge
+    links:
+      - "https://www.flyingcat.dance/gigs/newcontras"
+      - "https://www.facebook.com/events/3312637989021793"
+    start: 2022-10-22T19:45:00+01:00
+    end: 2022-10-22T23:00:00+01:00
+    country: UK
+    city: Cambridge
+    styles: [contra]
+    workshop: false
+    social: true
+    bands:
+      - Flying Cat
+      - Matt Cudby
+      - 'Flying Matt'
```

Comment:
Are these actually all one band, or three different bands playing separately?

Reply:
One band, playing under "Flying Matt", but I want "Flying Cat" to get their credit as I expect "Flying Matt" to be a one-time thing.